scripts/post-promote-tasks-utils.ts

```diff
@@ -0,0 +1,33 @@
+import fetch from 'node-fetch';
+
+interface IndexableVersions {
+  [channel: string]: string;
+}
+
+interface VersionDiff {
+  channel: string;
+  'amp-version': string;
+}
+
+export async function getVersionDiff(
+  head: string,
+  base: string
+): Promise<VersionDiff[]> {
+  const url = (sha: string) =>
+    `https://raw.githubusercontent.com/ampproject/cdn-configuration/${sha}/configs/versions.json`;
+
+  const headResponse = await fetch(url(head));
+  const headVersions = (await headResponse.json()) as IndexableVersions;
```

Comment:
Kinda wish there was a better way to tell the code "I want this JSON and I know its structure is going to be `ThisType`" 🙄